config/prod.exs

```diff
@@ -14,3 +14,8 @@ config :course_planner, CoursePlanner.Repo,
   url: System.get_env("DATABASE_URL"),
   pool_size: String.to_integer(System.get_env("POOL_SIZE") || "10"),
   ssl: true
+
+config :course_planner, CoursePlanner.Mailer,
+  adapter: Bamboo.MailgunAdapter,
+  api_key: System.get_env("MAILGUN_API_KEY"),
+  domain: "sandbox2c32a5cb470c4c3480c227b8ed604f75.mailgun.org"
```

Comment:
I believe we can leave like that until we actually have a separate staging environment.
Also we can have staging.ex as a separate config and keep there, no need to have it in a environment variable.